src/transformers/utils/quantization_config.py

```diff
@@ -456,10 +456,10 @@ def post_init(self):
             raise TypeError("bnb_4bit_use_double_quant must be a boolean")
 
         if self.load_in_4bit and not version.parse(importlib.metadata.version("bitsandbytes")) >= version.parse(
-            "0.39.0"
+            "0.43.3"
         ):
             raise ValueError(
-                "4 bit quantization requires bitsandbytes>=0.39.0 - please upgrade your bitsandbytes version"
+                "4 bit quantization requires bitsandbytes>=0.43.3 - please upgrade your bitsandbytes version"
```

Comment:
I'm not too keen on raising the min version for bnb. This might break a lot of workflow potentially. cc @Titus-von-Koeller

Reply:
Agreed. Let's keep this the same. If we do anything here, I think the way to go would be to instead handle this in `is_serializable` and/or `create_quantized_param` when the storage type is anything other than uint8 and the version is <0.43.3.